cachi2/core/package_managers/rpm/main.py

```diff
@@ -4,7 +4,7 @@
 import shlex
 from os import PathLike
 from pathlib import Path
-from typing import Any, Union
+from typing import Any, List, Union
```

Comment:
Whereas I don't oppose this change per se, I think we've been consistently using `list` for typing.

Reply:
My bad then for not checking the precedent then. While I think we should be consistent in using the types from typing that would result in an unrelated patch series, so I'll drop this hunk.